src/Microsoft.ML/Runtime/EntryPoints/MacroUtils.cs

```diff
@@ -197,10 +197,10 @@ public static T TrainerKindApiValue<T>(TrainerKinds trainerKind)
 
         public static bool IsTrainerOfKind(Type type, TrainerKinds trainerKind)
         {
-            if (type == typeof(Trainers.BinaryLogisticRegressor))
+            if (type == typeof(Trainers.LogisticRegressionBinaryClassifier))
```

Comment:
>LogisticRegressionBinaryClassifier [](start = 40, length = 34)

Thanks for fixing this Eric!
Actually, lines 200-204 are now not needed, they were there before only because the name for logistic classifiers didn't match the pattern of the other trainers (that the entry point names end with "Classifier" or "BinaryClassifier").

Reply:
Can you make a follow-up for that change?  The goal here is to unblock the build.  I don't want to add anymore risk.